spec/helpers/navigation_helper_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-class DummyView
+class NavigationHelper::DummyView
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.